src/promptflow-evals/promptflow/evals/evaluators/_eci/_eci.py

```diff
@@ -65,8 +65,8 @@ class ECIEvaluator:
     .. code-block:: python
 
         {
-            "label": "False",
-            "reasoning": "Some reason."
+            "ECI_score": "False",
```

Comment:
Hmmm, we call it a score, but the value is boolean?

Reply:
This is aligning with the spec, I'm neutral on changing this - on one hand. It's consistent with other evaluators. On the other hand, it's odd. Minsoo was also neutral on this.

Reply:
Reverted back to label.